torch/_torch_docs.py

```diff
@@ -4220,6 +4220,28 @@
 
 """)
 
+add_docstr(torch.randn_like,
+           r"""
+randn_like(input, dtype=None, layout=None, device=None, requires_grad=False) -> Tensor
+
+Returns a tensor with the same size as :attr:`input` that is filled with
+random numbers from a normal distribution with zero mean and variance of
+one. This is equivalent to
+``torch.randn(input.size(), dtype=dtype, layout=layout, device=device, requires_grad=requires_grad)``.
```

Comment:
don't those other parameters need input. prefixes?

Reply:
Hmm I meant to write that ``torch.randn_like(input, dtype, layout, device, requires_grad)`` is equivalent to the expression above. Will change.

Reply:
On second thought, I like your idea better so that the user can understand the default behavior more quickly.